mlir/lib/Conversion/VectorToXeGPU/VectorToXeGPU.cpp

```diff
@@ -165,6 +496,20 @@ struct TransferReadLowering : public OpRewritePattern<vector::TransferReadOp> {
     if (failed(transferPreconditions(rewriter, readOp)))
       return failure();
 
+    auto chip = xegpu::getXeGPUChipStr(readOp);
+    if ( chip != "pvc" && chip != "bmg") {
+      // perform additional checks -
+      if (failed(extraCheckForScatteredLoadStore(readOp, rewriter)))
```

Comment:
it seems the extracCheck is not actual extra check. it is a specific check about the compatiblity to load_gather, am I right?